discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/QueryNoticesOptions.java

```diff
@@ -336,6 +399,10 @@ public String filter() {
   /**
    * Gets the query.
    *
+   * A query search returns all documents in your data set with full enrichments and full text, but with the most
+   * relevant documents listed first. Use a query search when you want to find the most relevant search results. You
+   * cannot use `natural_language_query` and `query` at the same time.
+   *
```

Comment:
If we were writing this SDK by hand, we'd validate that `natural_language_query` and `query` are not both specified when constructing the options instance from the builder.  There's no support for this in the generator today but we should consider adding it.